tests/integration/imagerepo_and_policy_test.go

```diff
@@ -28,15 +28,15 @@ import (
 	imagev1 "github.com/fluxcd/image-reflector-controller/api/v1beta2"
 )
 
-func TestImageRepositoryScan(t *testing.T) {
+func TestImageRepositoryScanAndPolicyDigest(t *testing.T) {
```

Comment:
I thought about creating a separate file `imagepolicy_test.go` but it would require most of this test to be copied anyway so I went with just extending this one